src/utils/apiRequests/getAllProjects.ts

```diff
@@ -0,0 +1,22 @@
+import getsessionId from './getSessionId';
+import getStoredCurrency from '../countryCurrency/getStoredCurrency';
+
+export async function getAllProjects() {
```

Comment:
All these API call functions are having mostly the same code which can be abstracted.

Reply:
Ideally when all APIs are in place we will abstract this code to create one function for get, post, put requests accordingly

Reply:
I already build an abstraction for you to check at https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/297

Reply:
Thanks a ton !